### DISCUSSION TITLE
Shapefiles for ArcPro

### DISCUSSION THREAD

**Discussion by flh2** [gwlucastrig/Tinfour#93] [Q&A]
Hi!
I am trying to use an ArcPro shapefile as Bathymytry for the SVM model. I have converted the contours of my bathymytry and tried it as point data. The files from the arc pro save shapefiles as multiple files but the program needs both the shp and dbf to work. How can i load both of these into my properties file with .shp extension. I tried it as a zip and it could not read it. 

Any help would be hugely appreciated, It is probably my lack of knowledge of programing. 

A strugling dissertation student

**Comment by gwlucastrig** [comment]
Hi,

Very pleased to hear from you.

You should put all the shapefile files (dbf, shp, prj, etc) in the same
folder. For the SVM properties file,just refer to the .shp file.  The app
knows enough to look for the "sidecar" files.

There is some information on the page on "Using the Simple Volumetric
Model"
https://gwlucastrig.github.io/TinfourDocs/HydrographySimpleVolumetricModel

If can put your stuff somewhere i can access it, I would be glad to take a
look.

Gary

**Comment by gwlucastrig** [comment]
The information from the -inspect command is often helpful. If you email me
a copy of its output, I can take a look at it tonight.

**Comment by gwlucastrig** [comment]
I just pushed out a code change that might be related to your problem.  Under a very particular set of conditions, the VertexReaderShapefile class could mess up the signs of depth values.  This only happened when treating contours from a polygon or line based shapefile as depth values.  The original code was designed to use point features rather than polyline features.   However, it could extract point-values from polygon/line features.  But, if you were using the "bathymetryModel = depth" option and specifying a attribute as the source of depth values, you might see incorrect signs on input point values.   
  
Unfortunately, I am not close to pushing out a new release.  Do you have the ability to build your own Tinfour Jar files?    If not, I could probably send you one if you have some kind of file-sharing system at your disposal.

Gary

P.S.  My offer to look at the inspection results from your source data and properties files you are using for your project is still open if it would help.
